tests/test_solvermode/test_methods.py

```diff
@@ -5,6 +5,7 @@
 @pytest.mark.quick
 @pytest.mark.setup
 @pytest.mark.fluent_version("latest")
+@pytest.mark.skip("Too sensitive to settings API")
```

Comment:
Don't think this test is important for pyfluent. At some point, we shall remove some of the solvermode tests, which are basically testing specific settings in the settings API.